Gir.toml

```diff
@@ -426,6 +426,7 @@ status = "generate"
 name = "Gtk.Application"
 status = "generate"
 trait_name = "GtkApplicationExt"
+subclassing = true
```

Comment:
@sdroege This options now not used, can we delete it later?

Reply:
I guess we can delete it now already :)

Reply:
Ok, I will do it.
@alatiera Please remove this line

Reply:
done :)